src/plugins/lua/luaplugin.h

```diff
@@ -45,23 +47,12 @@ class LuaTableWriter;
 /**
  * This plugin allows exporting maps as Lua files.
  */
-class LUASHARED_EXPORT LuaPlugin : public Tiled::WritableMapFormat
+class LuaUtilWriter
 {
-    Q_OBJECT
-    Q_PLUGIN_METADATA(IID "org.mapeditor.MapFormat" FILE "plugin.json")
-
 public:
-    LuaPlugin();
-
-    bool write(const Tiled::Map *map, const QString &fileName) override;
-    QString nameFilter() const override;
-    QString shortName() const override;
-    QString errorString() const override;
-
-private:
     void writeMap(LuaTableWriter &, const Tiled::Map *);
     void writeProperties(LuaTableWriter &, const Tiled::Properties &);
-    void writeTileset(LuaTableWriter &, const Tiled::Tileset *, unsigned firstGid);
+    void writeTileset(LuaTableWriter &, const Tiled::Tileset &, unsigned firstGid, bool standalone=true);
```

Comment:
Coding style: please put spaces around the `=`.

Reply:
Done